packages/terra-layout/src/utilities.js

```diff
@@ -0,0 +1,31 @@
+import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
```

Comment:
If we ever wanted to inverse this, instead of going from css modules export to JS, we could make the breakpoints a json file and go from JSON to css with https://github.com/Updater/node-sass-json-importer

Not sure if this would help with the issues you were seeing when testing with jest.